resource/item.go

```diff
@@ -24,11 +24,16 @@ type Item struct {
 
 // ItemList represents a list of items
 type ItemList struct {
+	// Results defines the total number of items in the collection matching the current
+	// context without any filter applied. If the storage handler cannot compute this value, -1 is set.
+	Results int
```

Comment:
yes it does not belong to this commit. this was a mistake. but I would like to keep it since I've also extended `ItemList`.

`Results` is basically the total count of items fetched by mgo with any query params like limit and/or offset applied. and total is now the total count of items without any filters applied.
if someone wants to get meta information with the response (see https://github.com/rs/rest-layer/#custom-response-formatter--sender) it looks like this:

"meta": {"offset": 2,"page": 1,"results": 1,"total": 4}